### PR TITLE
Fund_db api  Postgres updates

### PR DESCRIPTION
**Pull request by Adam-W1**
https://digital.dclg.gov.uk/jira/browse/FS-2628

### Change description
The fund store is now using a Postgres DB, that has changed the way in which data is returned from the fund_store api that this microservice consumes. The changes in this PR adapt to consume the new fund_store responses.

- [x] Unit tests and other appropriate tests added or updated
- [x] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
pytest
### Screenshots of UI changes (if applicable)
NA

### CODE REVIEW

**Review by srh-sloan — APPROVED** [review]
LGTM

**Review by AbuNavsa — APPROVED** [review]

**Review by RamuniN — APPROVED** [review]
👍

File: app/assess/models/fund_summary.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def create_fund_summaries(fund: Fund) -> list[FundSummary]:
 def is_after_today(date_str: str):
     """Check if the provided datetime string has passed the current datetime"""
     uk_tz = pytz.timezone("Europe/London")
-    date_format = "%Y-%m-%d %H:%M:%S"
+    date_format = "%Y-%m-%dT%H:%M:%S"
```
Comment (aaronwilliamsv1) [inline app/assess/models/fund_summary.py:58]: This comes out as 2023-05-16T09:31:17 - is there a reason for the T? I don't see the output improvement

File: config/envs/default.py [hunk 2]
```diff
@@ -141,12 +141,12 @@ class DefaultConfig:
 
     TALISMAN_SETTINGS = CommonConfig.TALISMAN_SETTINGS
 
-    COF_FUND_ID = CommonConfig.COF_FUND_ID
-    COF_ROUND2_ID = CommonConfig.COF_ROUND_2_ID
-    COF_ROUND2_W3_ID = CommonConfig.COF_ROUND_2_W3_ID
+    COF_FUND_ID = "47aef2f5-3fcb-4d45-acb5-f0152b5f03c4"
```
Comment (aaronwilliamsv1) [inline config/envs/default.py:144]: Why are these being added here? Why not update https://github.com/communitiesuk/funding-service-design-utils/blob/main/fsd_utils/config/commonconfig.py?

File: requirements-dev.txt [hunk 3]
```diff
@@ -167,7 +167,7 @@ flipper-client==1.3.2
     # via
     #   -r requirements.txt
     #   funding-service-design-utils
-funding-service-design-utils==1.0.28
+funding-service-design-utils @ git+https://github.com/communitiesuk/funding-service-design-utils.git@fund-store-db
```
Comment (aaronwilliamsv1) [inline requirements-dev.txt:170]: This feels a temporary change that shouldn't be commited

File: requirements.in [hunk 4]
```diff
@@ -1,7 +1,7 @@
 #-----------------------------------
 #   FSD Utils
 #-----------------------------------
-funding-service-design-utils>=1.0.28,<1.1.0
+git+https://github.com/communitiesuk/funding-service-design-utils.git@fund-store-db
```
Comment (aaronwilliamsv1) [inline requirements.in:4]: Feels like a temporary change that shouldn't be committed

File: requirements.txt [hunk 5]
```diff
@@ -91,7 +91,7 @@ flask-wtf==1.1.1
     # via -r requirements.in
 flipper-client==1.3.2
     # via funding-service-design-utils
-funding-service-design-utils==1.0.28
+funding-service-design-utils @ git+https://github.com/communitiesuk/funding-service-design-utils.git@fund-store-db
```
Comment (aaronwilliamsv1) [inline requirements.txt:94]: Feels like a temporary change that shouldn't be committed

File: app/assess/models/fund_summary.py (hunk 1 above)
Comment (Adam-W1) [reply app/assess/models/fund_summary.py:58]: Its ISO 8601 format: https://en.wikipedia.org/wiki/ISO_8601

File: config/envs/default.py (hunk 2 above)
Comment (Adam-W1) [reply config/envs/default.py:144]: All fund info will be in the fund store, this is a bit of a shortcut whilst that change goes in but in the long term we shouldn't be hardcoding funds at all (hence a call to COF_FUND_ID will disappear all together)

File: app/assess/models/fund_summary.py (hunk 1 above)
Comment (aaronwilliamsv1) [reply app/assess/models/fund_summary.py:58]: Ah, I see, it's being used as a calculation comparison. Works for me

File: requirements-dev.txt (hunk 3 above)
Comment (aaronwilliamsv1) [reply requirements-dev.txt:170]: I can't then resolve this until that PR is merged. For records sake: https://github.com/communitiesuk/funding-service-design-utils/pull/74 is the PR

**Review by gio-karunakaran — APPROVED** [review]
👍🏾

**Pull request merged**